input/Shared/switches.txt

```diff
@@ -8,6 +8,8 @@
 | --showtree, --tree        | Shows the task dependency tree.                                                                    | [0.31.0]        |
 | --dryrun                  | Performs a dry run.                                                                                | [0.2.0]         |
 | --exclusive               | Execute a single task without any dependencies.                                                    | [0.29.0]        |
+| --bootstrapping           | Download/install modules defined by `#module` directives, but do not run build.                    | [0.24.0]        |
```

Comment:
Isn't this just `bootstrap`?

Reply:
Of course. Fixed

Reply:
Also, I think I left this out of the original list that I put together, due to the fact that this is being removed, unless we add another column to say when it was removed.

Reply:
It hasen't been removed

Reply:
Why is it needed?  This happens automatically now?

Reply:
I was not really involved with this, just stumbled across this as I tried to update CLI switch documentation. Explicit bootstrapping was done with https://github.com/cake-build/cake/issues/2833. Maybe because we didn't want to break stuff?  @patriksvensson or @devlead might have more details.

Reply:
As discussed with @devlead :

> To not break every existing bootstrapper script/github action/azure pipelines extension etc.
It still works and does what it did before.
If we remove it now it would potentially mean scripts execute twice and that would be bad.